### PR TITLE
Website and faq update

### PR DESCRIPTION
**Pull request by lindaicing**
Screen Shot 2020-09-28 at 12 33 25 AM
Screen Shot 2020-09-28 at 12 33 40 AM
Screen Shot 2020-09-28 at 12 34 10 AM

### CODE REVIEW

File: Website-and-Backend/src/main/resources/templates/index.html [hunk 1]
```diff
@@ -125,6 +125,24 @@ <h3>Info Page</h3>
...
+            <section>
+                <article>
+                    <h3>Girlfriend Collective</h3>
+                    <span>
+                        <img src="https://cdn.shopify.com/s/files/1/0019/2217/0943/products/1_5f385713-3fe7-4c44-b4f2-1e57eea1b41d_1200x.jpg?v=1598520929" alt="Girlfriend Merch">
+                    </span>
+                    <p>We're partnering with Girlfriend Collective! We love Girlfriend Collective for their model diversity and extensive circular program 
+                        where they buy back their used items (made from recycled BPA-free water bottles) to make new ones! 
+                        They operate in a SA8000-certified zero-waste facility, are size-inclusive (XXS—6XL), anti-child labor, and vegan.</p>
+                    <p class="readmore"><a href="https://www.girlfriend.com" target="_blank">Visit Website</a></p>
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/templates/index.html:140]: Use our affiliate link here

File: Website-and-Backend/src/main/resources/templates/index.html [hunk 2]
```diff
@@ -125,6 +125,24 @@ <h3>Info Page</h3>
...
+            <h2><span class="colorCream">Partners</span></h2>
+            <section>
+                <article>
+                    <h3>Girlfriend Collective</h3>
+                    <span>
+                        <img src="https://cdn.shopify.com/s/files/1/0019/2217/0943/products/1_5f385713-3fe7-4c44-b4f2-1e57eea1b41d_1200x.jpg?v=1598520929" alt="Girlfriend Merch">
+                    </span>
+                    <p>We're partnering with Girlfriend Collective! We love Girlfriend Collective for their model diversity and extensive circular program 
+                        where they buy back their used items (made from recycled BPA-free water bottles) to make new ones! 
+                        They operate in a SA8000-certified zero-waste facility, are size-inclusive (XXS—6XL), anti-child labor, and vegan.</p>
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/templates/index.html:139]: @amtims Does this text look good to you?

**Review by CubeOfCheese — CHANGES_REQUESTED** [review]

**Comment by CubeOfCheese** [comment]
Since we only have one, should it fill up the whole width?

File: Website-and-Backend/src/main/resources/templates/index.html (hunk 2 above)
Comment (amtims) [reply Website-and-Backend/src/main/resources/templates/index.html:139]: We're partnering with Girlfriend Collective! We _(**love --> appreciate**) (**Girlfriend Collective --> this brand**)_ for their model diversity and extensive circular program _(**where they buy back -->that buys back**)_ their used items (made from recycled BPA-free water bottles) to make new ones! They operate in a SA8000-certified zero-waste facility, are size-inclusive (XXS—6XL), anti-child labor, and vegan.

**Comment by lindaicing** [comment]
> Since we only have one, should it fill up the whole width?

Full width doesn't look as good at a large size, unless you want a different layout? @CubeOfCheese

File: Website-and-Backend/src/main/resources/templates/index.html (hunk 2 above)
Comment (lindaicing) [reply Website-and-Backend/src/main/resources/templates/index.html:139]: Thank you @amtims! Updated

**Comment by CubeOfCheese** [comment]
> > Since we only have one, should it fill up the whole width?
> 
> Full width doesn't look as good at a large size, unless you want a different layout? @CubeOfCheese

I just thought all the empty space looked a little odd, but it looks good regardless! I say let's keep it how it is

**Review by CubeOfCheese — APPROVED** [review]

**Comment by lindaicing** [comment]
> > > Since we only have one, should it fill up the whole width?
> > 
> > 
> > Full width doesn't look as good at a large size, unless you want a different layout? @CubeOfCheese
> 
> I just thought all the empty space looked a little odd, but it looks good regardless! I say let's keep it how it is

okiedoke

**Pull request merged**